MA-FE-NC-GAMES/components/CommentAdder.jsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from "react";
+import { postComment } from "./api";
+
+const CommentAdder = ({ review_id, setComments, values, setValues }) => {
+	const [err, setErr] = useState(null);
+
+	const handleInputChange = (event) => {
+		setValues({
+			username: "jessjelly",
+			body: event.target.value,
+			comment_id: 10000,
+			author: "jessjelly",
+			votes: 0,
+			created_at: new Date().toISOString(),
+		});
+	};
+
+	const handleSubmit = (event) => {
+		event.preventDefault();
+		setComments((currentComments) => {
+			return [values, ...currentComments];
+		});
+		postComment(review_id, values)
+			.then((res) => {
+				setValues({
+					username: "jessjelly",
+					body: "",
+				});
+				return res;
+			})
+			.catch((err) => {
+				setErr(
+					`${err.response.data.msg}. Please refresh the page and try again.`
+				);
+			});
+	};
+
+	return (
+		<>
+			<form onSubmit={handleSubmit}>
```

Comment:
We want to add some form validation here to prevent us being able to post an empty comment :)